EverestAPI/Services/ICustomerService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using EverestAPI.Models;
+
+namespace EverestAPI.Services
+{
+    public interface ICustomerService
+    {
+        void Create(CustomerModel model);
```

Comment:
```suggestion
        Long Create(CustomerModel model);
```
Deveria retornar o Id do Customer que acabou de ser criado. PS: Long foi só uma das hipóteses. Poderia ser int, Guid e etc.